app/javascript/components/pages/ApartmentNew.js

```diff
@@ -24,6 +24,7 @@ const ApartmentNew = ({ createApartment, logged_in, current_user }) => {
     newApartment.user_id = current_user.id
     createApartment(newApartment)
     navigate("/apartmentindex")
+    document.location.reload()
```

Comment:
I think we should explore other options to render apartment index than manipulating the dom directly.